sdk/src/main/java/com/mapbox/maps/Style.kt

```diff
@@ -963,6 +963,22 @@ class Style internal constructor(
     return styleManager.setStyleSourceProperties(sourceId, properties)
   }
 
+  override fun setStyleSourceData(sourceId: String, data: Value): Expected<String, None> {
+    // could be called from any thread
```

Comment:
There is [AnyThreaf](https://developer.android.com/reference/kotlin/androidx/annotation/AnyThread) annotation, should we consider marking our APIs accordingly?

Reply:
yeah, perhaps - but when doing so it makes sense guess to update all other methods with `@UiThread` which is obviously not in scope of this PR. Will give it a thought, thanks 😄